packages/shaderc.rb

```diff
@@ -0,0 +1,55 @@
+# Adapted from Arch Linux shaderc PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/shaderc/trunk/PKGBUILD
+
+require 'package'
+
+class Shaderc < Package
+  description 'Collection of tools, libraries and tests for shader compilation'
+  version '2020.5'
+  license 'Apache'
+  compatibility 'all'
+  source_url "https://github.com/google/shaderc/archive/refs/tags/v#{version}.tar.gz"
+  source_sha256 'e96d8cb208b796cecb9e6cce437c7d1116343158ef3ea26277eb13b62cf56834'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/shaderc-2020.5-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/shaderc-2020.5-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/shaderc-2020.5-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/shaderc-2020.5-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '74e0264a8277b173f33e04e84a31a118173c2f4c5013ddd9f68f51a4a38b0965',
+     armv7l: '74e0264a8277b173f33e04e84a31a118173c2f4c5013ddd9f68f51a4a38b0965',
+       i686: '061996cc65ec0c60bd26a09b26e5d949504d2c6718eae3531949b4ccf9d2da3f',
+     x86_64: '9e32e0db031138d5ead88e9e6a99f1a14d1d07e6c89999f0d336d105462b1306'
+  })
+
+  @asciidoctor_installed = `gem list | grep ^asciidoctor`
+
+  def self.build
+    system 'gem install -N asciidoctor' unless @asciidoctor_installed.to_s != ''
```

Comment:
Should we use `depends_on 'asciidoctor'` instead?

Reply:
Yup.